sonic_platform_base/sonic_sfp/sfputilhelper.py

```diff
@@ -45,6 +47,22 @@ class SfpUtilHelper(object):
     def __init__(self):
         pass
 
+    def get_port_mapping(self):
```

Comment:
MultiASIC is not taken into consideration for the original approach with Config DB